ios/native/REAIOSScheduler.h

```diff
@@ -13,7 +13,6 @@ class REAIOSScheduler : public Scheduler {
  public:
   REAIOSScheduler(std::shared_ptr<CallInvoker> jsInvoker);
   void scheduleOnUI(std::function<void()> job) override;
-  virtual ~REAIOSScheduler();
```

Comment:
`REAIOSScheduler` extends `Scheduler` which has virtual methods so it needs to have a virtual destructor as well. We can move the implementation from .cpp to .h file if possible.